engine_json_features.ipynb

```diff
@@ -4,9 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Howso Engine &#x2014; JSON/YAML as Features\n",
+    "# JSON/YAML Features with Howso Engine\n",
     "\n",
-    "#### Note - This feature is experimental and is not universally recommended for production deployment.\n",
+    "> Note: This feature is experimental and is not universally recommended for production deployment.\n",
```

Comment:
Should we mark as beta?

Reply:
This almost feels pre-beta... Have we every called anything "Alpha"?

Reply:
Not that I know of, but I think it would be acceptable if no one else has issues with it.